acceptance/suites/pre_suite/upgrade/10_install_puppet.rb

```diff
@@ -1,5 +1,5 @@
-step "Install PC1 repository" do
-  install_puppetlabs_release_repo(hosts, 'pc1')
+step "Install puppet repository" do
+  install_puppetlabs_release_repo(hosts, 'puppet')
```

Comment:
@melissa I don't know what specific blockers we might have in moving to the latest beaker version.  We'd probably just have to try it out and make changes where necessary for anything that might break.  I could submit a separate ticket for us to investigate that and we can block this PR the resolution of the new ticket, if that makes sense to you.  WDYT?